src/python/aqueduct_executor/operators/connectors/data/snowflake.py

```diff
@@ -4,11 +4,11 @@
 
 class SnowflakeConnector(relational.RelationalConnector):
     def __init__(self, config: config.SnowflakeConfig):
-        conn_engine = _create_engine(config)
+        conn_engine = create_snowflake_engine(config)
         super().__init__(conn_engine)
 
 
-def _create_engine(config: config.SnowflakeConfig) -> engine.Engine:
+def create_snowflake_engine(config: config.SnowflakeConfig) -> engine.Engine:
```

Comment:
Doesn't look like this is imported anywhere. Is there a reason for the name change?